README.md

```diff
@@ -364,6 +365,56 @@ Main | [Info] This is an informational message.
 Main | [Warn] This is a warning, should take a look.
 ```
 
+## `LevelOverrideLogger` (*Filter*)
+Allows overriding the log level set by any nested logger. Useful when debug logging
+and used in conjuction with `Logging.with_logger` or `LoggingExtras.with` to
+temporarily modify the current logger with a custom level.
+
+```julia
+julia> using LoggingExtras
+
+julia> logger = LevelOverrideLogger(Debug, global_logger())
+
+julia> with_logger(logger) do
+           @debug "This message will log since we're overriding the global Info default log level"
+       end
+┌ Debug: This message will log since we're overriding the global Info default log level
+└ @ Main REPL[33]:2
+```
+
+# Utilities
+
+## Verbosity macros
```

Comment:
Somewhere in here need to mention that `verbosity` is just an abstraction built on storing an integer in the `group`.

Reply:
No longer true so resolving